integration_tests/tests/apply/apply.cy.ts

```diff
@@ -325,6 +327,26 @@ context('Apply', () => {
       const oasysSelection = [...oasysSectionsLinkedToReoffending, ...otherOasysSections]
       cy.task('stubOasysSelection', { person, oasysSelection })
 
+      const roshSummarySectionA = roshSummaryFactory.build({
+        questionNumber: '1',
+        label: 'The first question',
+        answer: 'Some answer for the first question',
+      })
+      const roshSummarySectionB = roshSummaryFactory.build({
+        questionNumber: '2',
+        label: 'The second question',
+        answer: 'Some answer for the second question',
+      })
+      const roshSummarySectionC = roshSummaryFactory.build({
+        questionNumber: '3',
+        label: 'The third question',
+        answer: 'Some answer for the third question',
+      })
+      const oasysSections = oasysSectionsFactory.build({
+        roshSummary: [roshSummarySectionA, roshSummarySectionB, roshSummarySectionC],
+      })
```

Comment:
I'm a bit iffy about this way of building objects as we're relying on the data in the fixtures to be the same, and the tests will break if we change if one of the other places. I added a `documentsFromApplication` helper here https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/main/integration_tests/support/helpers.ts#L4, so could do something similar for the summaries.

Reply:
To ensure the data that comes back from the APIs is different from the user-inputted data, you could remove the `With an extra comment` parts from the stub objects too.

Reply:
Yeah I think this is a nice improvement. It does rely on us having an unmodified copy of the data in `applicationData` or not modifying the data in the application but that's fine for these pages